server/frontend/src/components/Planets.js

```diff
@@ -95,13 +108,16 @@ class Planets extends Component {
                height={450}
                headerHeight={90}
                rowHeight={40}
-               rowCount={this.state.planets.length}
-               rowGetter={({ index }) => this.state.planets[index]}
+               rowCount={this.state.planets.size}
+               rowGetter={({ index }) => this.state.planets.get(index)}
                rowClassName={this._rowClassName}
                onRowClick={(props) => this.showMoons(props.rowData)}
+               sort={this._sort}
+               sortBy={sortBy}
+               sortDirection={sortDirection}
                >
-          <Column label='Name' dataKey='name' width={70} className='main-column' />
-          <Column label='Moons' dataKey='numberOfMoons' width={70} />
+          <Column label='Name' dataKey='name' disableSort={false} width={70} className='main-column' />
```

Comment:
Why do we need `disableSort={false}` for these two columns? It seems to be the default. Sorting works for me in the same way without this two lines changed.

Reply:
Yes, it works without this argument. I think we can remove it.